keyboards/decent/numpad/keyboard.json

```diff
@@ -0,0 +1,150 @@
+{
+    "manufacturer": "Bertrand Le roy",
+    "keyboard_name": "DecenTKL NumPad",
+    "maintainer": "bleroy",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "encoder": false,
+        "bootmagic": true,
+        "rgb_matrix": true,
+        "oled": false,
+        "command": false,
+        "console": false,
+        "extrakey": false,
+        "mousekey": false,
+        "nkro": false
```

Comment:
```suggestion
        "bootmagic": true,
        "rgb_matrix": true,
        "extrakey": true,
        "mousekey": true
```

Reply:
Extra keys are stuff like volume control, media control, and the like.   For the configurator and via, people may expect it to be enabled by default.
As for mousekeys, similar, but I would argue, not as important.

Reply:
You can choose not to enable them (extra keys/mouse keys) if you deem them really unnecessary, but the `false` options should be removed from the `features` keys regardless.